romanesco/format/__init__.py

```diff
@@ -99,53 +181,38 @@ def import_converters(search_paths):
                     "url": analysis["script_uri"]
                 })
 
-            if os.path.basename(filename).startswith("validate_"):
-
-                # This is a validator
-                in_type = analysis["inputs"][0]["type"]
-                in_format = analysis["inputs"][0]["format"]
-                if in_type not in validators:
-                    validators[in_type] = {}
-                validators[in_type][in_format] = analysis
-
-            else:
-
-                # This is a converter
-                in_type = analysis["inputs"][0]["type"]
-                in_format = analysis["inputs"][0]["format"]
-                out_format = analysis["outputs"][0]["format"]
-                if in_type not in converters:
-                    converters[in_type] = {}
-                analysis_type = converters[in_type]
-                if in_format not in analysis_type:
-                    analysis_type[in_format] = {}
-                input_format = analysis_type[in_format]
-                if out_format not in input_format:
-                    input_format[out_format] = {}
-                input_format[out_format] = [analysis]
+        return analysis
 
-    os.chdir(prevdir)
+    prevdir = os.getcwd()
+    for path in search_paths:
+        os.chdir(path)
+        validator_files = set(glob.glob(os.path.join(path, "validate_*.json")))
+        converter_files = set(glob.glob(os.path.join(path, "*.json"))) - validator_files
```

Comment:
I think a better convention would be to change `*.json` to `*_to_*.json, and still subtract `validator_files` on the off chance some format name contains `_to_`.